qa/tasks/cephfs/test_openfiletable.py

```diff
@@ -44,8 +44,8 @@ def test_max_items_per_obj(self):
 
         self.fs.radosm(["stat", "mds0_openfiles.1"])
 
-        # Now close the file
-        self.mount_a.kill_background(p)
+        # Now close the files
+        self.mount_a.kill_background()
```

Comment:
I like this change but note that `CephFSTestCase.tearDown` immediately calls `m.teardown()` for each mount which kills all background jobs.

So I don't think this is fixing anything? What concerns me now that I'm looking at this more closely again is that `CephFSTestCase.tearDown` does not actually unmount any mount before deleting file systems... ???

Reply:
(Perhaps the test infrastructure was relying on the `setUp` to unmount any existing mounts but that is very wrong...)

Reply:
Yeah, the `tearDown` will help kill all the background, but that seems a little late.

@batrick When killing the background opened file I can see the following logs, which was added by you in the `teuthology` repo:

`
2024-03-31T22:51:40.067 INFO:teuthology.orchestra.run.smithi136.stderr:2024-03-31T22:51:40 stdin-killer INFO: expiration expected; waiting 5 seconds for command to complete
`
Does that means it will be stuck for 5s when closing the files to be closed ? Or should the unmount also need to wait for 5s before doing real mount ?

Reply:
See #56638

I think we should just remove the `kill_background` calls in this test case. It should be handled by `tearDown`.